index.js

```diff
@@ -0,0 +1,87 @@
+import speaker from './module/data.js';
+
+const guestSpeacker = document.querySelector('#guest');
+// Mobile menue ------------
+const hamburger = document.querySelector('.hamburger');
+const mobileLink = document.querySelectorAll('.mobile-link');
+const mobileMenue = document.querySelector('.mobile-menu');
+const seeMoreBtn = document.querySelector('.seemore');
+function mobileToggleClassActive() {
+  hamburger.classList.toggle('active');
+  mobileMenue.classList.toggle('active');
+}
+
+hamburger.addEventListener('click', mobileToggleClassActive);
+mobileLink.forEach((element) => {
+  element.addEventListener('click', mobileToggleClassActive);
+});
+
+guestSpeacker.innerHTML = `
+<h2 class="feature-h2">Featured Speakers</h2>
+    <hr class="feature-hr">
+    <div class="feature-container">
+    
+      <div >
+        <div class="guestSpeaker">
+          <div class="guestSpeaker-img">
+            <img src="${speaker[2].image}" class="quest-img" alt="img">
+          </div>
+          <div class="content">
+            <h2 class="name">${speaker[2].name}</h2>
+            <p class="profession">${speaker[2].profession}</p>
+
+            <p class="about-me">${speaker[2].aboutMe}</p>
+          </div>
+        </div>
+        <div class="guestSpeaker last">
+          <div class="guestSpeaker-img">
+            <img src="${speaker[3].image}" class="quest-img" alt="img">
+          </div>
+          <div class="content">
+            <h2 class="name">${speaker[3].name}</h2>
+            <p class="profession">${speaker[3].profession} </p>
+
```

Comment:
Let's add more speakers to the speaker section so as to match the given design😊 You currently have four, let's add two more.

yours
<img width="899" alt="feature speakers" src="https://user-images.githubusercontent.com/64643995/225445817-bcafd1cc-6e53-45da-8f40-c3d5be762f18.png">

design
<img width="568" alt="speakers design" src="https://user-images.githubusercontent.com/64643995/225445851-66a923b6-9167-4850-b1aa-6601741cb4d7.png">